### PR TITLE
fix pr-comment workflow

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

**Comment by dpo** [comment]
@amontoison Does this fix it for you?

File: github-workflows/pr-comment.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ jobs:
     runs-on: [ubuntu-latest]
     steps:
       - name: add artifact links to pull request and related issues step
-        if: needs.pr-check.outputs.number != null
         uses: veitbjarsch/artifacts-url-comments@v1.1.0
```
Comment (amontoison) [inline github-workflows/pr-comment.yml:13]:
```suggestion
        uses: veitbjarsch/artifacts-url-comments@main
```

**Comment by amontoison** [comment]
> @amontoison Does this fix it for you?

@dpo No, but I found the culprit. The workflow is not working anymore because Node.js 12 is deprecated.
The workflow supports Node.js 16 but only on the branch `main`.

**Comment by dpo** [comment]
Thank you!

**Pull request merged**